internal_test.go

```diff
@@ -0,0 +1,35 @@
+package jsonrpc2
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestAnyMessage(t *testing.T) {
```

Comment:
This is the only test that was left in the `object_test.go` file after I moved `TestRequest_MarshalUnmarshalJSON` into `request_test.go` and `TestResponse_MarshalUnmarshalJSON` into `responde_test.go`. I then realized that this is the only internal test file (all other test files contain external tests). So with that realization and the fact that I did not really understand why this file was called `object_test.go` to begin with, I thought it was more intuitive if this file was named `internal_test.go`.